NEWS.md

```diff
@@ -1,30 +1,34 @@
 # scoringutils 1.3
 
-This major update introduces a lot of breaking changes and addresses comments made by review from the Journal of Statistical Software (see preprint of the manuscript [here](https://arxiv.org/abs/2205.07090)).
+This major update and addresses a variety of comments made by reviewers from the Journal of Statistical Software (see preprint of the manuscript [here](https://arxiv.org/abs/2205.07090)).
+
+The update introduces a lot of breaking changes. If you want to keep using the older version, you can download it using `remotes::install_github("epiforecasts/scoringutils@v1.2")`. 
```

Comment:
drop a lot

Reply:
These news changes looks like they are really for `rework-lower-level-fcts`. Mixing PRs makes reviewing very difficult and mentally taxing.

Reply:
Done - and apologies for the mix of things. I promise the PRs are getting better over time!